Putative Passenger tooltip icon

dianab0 opened the pull request:
There is a flash icon for the Putative Driver in the Oncoprint tooltip, but there is no icon for the Putative Passenger. I have added a gray circle icon for it as in the gif below.

![Putative-Passenger](https://user-images.githubusercontent.com/53996876/149132057-4b3ab352-dec4-44ce-b034-a0a1e2389432.gif)

alisman commented:
@dianab0 you could probably accomplish this with svg in about 1% of the file size.  here's a start:
```
<svg xmlns="http://www.w3.org/2000/svg" version="1.1">
   <circle cx="100" cy="50" r="40" stroke="black" stroke-width="2" fill="none" />
</svg> 
```

dianab0 commented:
Rebased to the new master and changed the img icon to svg circle as below:

![Screenshot from 2022-01-17 15-06-42](https://user-images.githubusercontent.com/53996876/149786091-5918b491-22cb-435e-9bfc-21d0308d025d.png)

pvannierop commented on src/shared/components/oncoprint/TooltipUtils.ts:
```diff
@@ -585,6 +585,15 @@ export function makeGeneticTrackTooltip(
                         );
                     }
+                    //If we have data for the binary custom driver annotations, append an icon to the tooltip with the annotation information
+                    if (driver_filter && driver_filter === PUTATIVE_PASSENGER) {
```
Small thing. This should be an `else if` since driver (previous condition) and passenger are mutually exclusive.

dianab0 replied:
Done

pvannierop reviewed (changes requested):
One small correction suggested.
